.github/workflows/_docker-build.yml

```diff
@@ -84,7 +84,7 @@ jobs:
 
   docker:
     name: 'docker'
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Maybe it only fails on ubuntu-24.04, see: https://github.com/actions/runner-images/issues/10636#issuecomment-2610539943